tests/GasModelSpec.hs

```diff
@@ -120,7 +120,7 @@ runTest t = mapOverGasUnitTests t run run
       return (res', st)
     setupEnv' dbs = do
       (r, s) <- setupEnv dbs
-      let r' = set eeExecutionConfig (mkExecutionConfig [FlagDisableInlineMemCheck]) r
+      let r' = set eeExecutionConfig (mkExecutionConfig [FlagDisableInlineMemCheck, FlagDisablePactEvents, FlagDisablePact43]) r
```

Comment:
sigh ... we need to be regressing new stuff too :(

Reply:
in a later PR